tests/jax/unit-tests-cc.libsonnet

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local common = import 'common.libsonnet';
+local mixins = import 'templates/mixins.libsonnet';
+
+{
+  local compilationCacheTest = common.JaxTest + mixins.Functional {
+    modelName: 'compilation-cache-test',
+
+    testScript:: |||
+      set -x
+      set -u
+      set -e
+
+      # .bash_logout sometimes causes a spurious bad exit code, remove it.
+      rm .bash_logout
+
+      pip install --upgrade pip
+      pip install --upgrade numpy==1.18.5 scipy wheel future six cython pytest \
+          absl-py opt-einsum msgpack
+
+      echo "Checking out and installing JAX..."
+      git clone https://github.com/colemanliyah/jax.git
+      cd jax
+      git fetch
+      git checkout pmap_integration
```

Comment:
Reminder (for both of us) to commit this until your pmap change is merged and we can checkout the main repo again.